experiments/ClimaEarth/components/ocean/oceananigans.jl

```diff
@@ -62,14 +74,25 @@ function OceananigansSimulation(area_fraction; output_dir, comms_ctx = ClimaComm
 
     grid = OC.ImmersedBoundaryGrid(underlying_grid, OC.GridFittedBottom(bottom_height); active_cells_map = true)
 
-    # Create ocean simulation
-    ocean = CO.ocean_simulation(grid)
+    if use_ecco
```

Comment:
Could you add an inline comment explaining that if we use ECCO, we have forcings and don't need to set ICs, otherwise we do need to set ICs?